ci_feature/setup.sh

```diff
@@ -14,7 +14,7 @@ wget https://github.com/Microsoft/OMS-Agent-for-Linux/releases/download/OMSAgent
 #create file to disable omi service startup script
 touch /etc/.omi_disable_service_control
 
-wget https://github.com/Microsoft/Docker-Provider/releases/download/4.0.0-0/docker-cimprov-4.0.0-0.universal.x86_64.sh
+wget https://dockerprovider.blob.core.windows.net/prometheus/docker-cimprov-5.0.0-0.universal.x86_64.sh
```

Comment:
Is the agent going to be released from this private storage account?

Reply:
this will change when we have a release build and move to ci_feature_prod. How else we test the build ? This is not new :)